### PR TITLE
Add Jira source

### PR DESCRIPTION
**Pull request by jaceksan**

### CODE REVIEW

**Review by Mara3l — APPROVED** [review]

File: data_pipeline/meltano-plugins.yml [hunk 1]
```diff
@@ -12,6 +12,9 @@ plugins:
     - name: tap-exchangeratehost
       variant: anelendata
       pip_url: git+https://github.com/anelendata/tap-exchangeratehost.git
+    - name: tap-jira
+      variant: meltanolabs
+      pip_url: git+https://github.com/jaceksan/tap-jira.git
```
Comment (hkad98) [inline data_pipeline/meltano-plugins.yml:17]: Consider adding commit sha or tag for the 1.0 version.

File: data_pipeline/meltano_conf/extractors/jira.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+plugins:
+  extractors:
+    - name: tap-jira
+      variant: meltanolabs
+      pip_url: git+https://github.com/jaceksan/tap-jira.git
+      config:
+        domain: gooddata.atlassian.net
+        auth:
+          flow: password
+          username: tech.demo.jira@gooddata.com
```
Comment (hkad98) [inline data_pipeline/meltano_conf/extractors/jira.yml:10]: Can we hide this in env variable?

**Review by hkad98 — CHANGES_REQUESTED** [review]

File: data_pipeline/gooddata_layouts/cicd_demo/analytics_model/analytical_dashboards/b1b599b2-388e-4cb8-ab97-f02fce480244.yaml [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+                  type: visualizationObject
+              properties: {}
+              title: Missed Due Date in History
+              type: insight
+        type: IDashboardLayoutSection
+    type: IDashboardLayout
+  version: '2'
+description: ''
+id: b1b599b2-388e-4cb8-ab97-f02fce480244
+permissions:
```
[the remark this answers is not in the document]
Comment (jaceksan) [reply data_pipeline/gooddata_layouts/cicd_demo/analytics_model/analytical_dashboards/b1b599b2-388e-4cb8-ab97-f02fce480244.yaml]: Removed

File: data_pipeline/meltano-plugins.yml (hunk 1 above)
Comment (jaceksan) [reply data_pipeline/meltano-plugins.yml:17]: Added a special branch.

I created PR to upstream repo. Once it's merged, I switch back to upstream.

File: data_pipeline/meltano_conf/extractors/jira.yml (hunk 2 above)
Comment (jaceksan) [reply data_pipeline/meltano_conf/extractors/jira.yml]: Hidden

**Review by hkad98 — APPROVED** [review]

**Pull request merged**